File: Python/Estrutura sequencial/troco.py
# Variáveis
# Em Python não é necessário fazer esse passo...
precoUnitario: float; precoTotal: float; dinheiroRecebido: float; troco: float
quantidade: int

# Entrada de dados e armazenamento de dados nas variáveis
precoUnitario = float(input("Preço unitário do produto: "))
quantidade = int(input("Quantidade comprada: "))
dinheiroRecebido = float(input("Dinheiro recebido: "))

# Artíficio matemático para calcular o troco
precoTotal = precoUnitario * quantidade
troco = dinheiroRecebido - precoTotal

# Saída de dados
print(f"O troco que o cliente deve receber é de R${troco}")

File: C++/Estrutura sequencial/troco.cpp
// Biblioteca
#include <bits/stdc++.h>

using namespace std;

int main()
{
    // Variáveis
    double precoUnitario, precoTotal, dinheiroRecebido, troco;
    int quantidade;

    // Entrada de dados
    cout << "Preco unitario do produto: ";
    cin >> precoUnitario;

    cout << "Quantidade comprada: ";
    cin >> quantidade;

    cout << "Dinheiro recebido: ";
    cin >> dinheiroRecebido;

    // Algoritmo matemático
    precoTotal = (double) precoUnitario * quantidade;
    troco = dinheiroRecebido - precoTotal;

    // Saída de dados
    cout << fixed << setprecision(2); // Quantidade de casas decimais
    cout << "O troco que o cliente deve receber e de R$" << troco;

    return 0;
}